client/coral-i18n/services/i18n.js

```diff
@@ -0,0 +1,93 @@
+const time = require('timeago.js');
+const has = require('lodash/has');
+const get = require('lodash/get');
+
+const esTA = require('../../../node_modules/timeago.js/locales/es');
+
+// Translations are happening at https://www.transifex.com/the-coral-project/talk-1/dashboard/.
+
+const defaultLanguage = 'en';
+
+let translations = {};
+let timeagoInstance;
+
+const fetchTranslations = (locale) => {
+  translations = require(`json-loader!yaml-loader!../locales/${locale}.yml`)[locale];
```

Comment:
This has no benefit over just importing all our current locals with simple static import statements. The fetchTranslations function shouldn't be needed as all translations should come from the bundle.